app/commands/decidim/proposals/admin/update_proposal.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Decidim
```

Comment:
ℹ️ Same as Mike's PR, these are copies from the Proposal gem, but using the Rails Engine override. I'll highlight the relevant changes....